Makefile.rules

```diff
@@ -428,6 +428,16 @@ ffdata: ffgen
 
 
 ########################################################################
+# The "tags" target regenerates ctags
+########################################################################
+
+tags:
+	cd $(abs_top_srcdir) && ctags --sort=no lib/*.g lib/*.gi lib/*.gd src/*
```

Comment:
Could we dinosaurs have support for emacs tags as well? Just run the same thing again with `-e F TAGS` or something of the kind.

Reply:
What does the "F"? in there do?

Problem is that on Mac OS, files `tags` and `TAGS` cannot coexist by default on HFS+ volumes...

But we could have two targets: `etags` and `ctags`

Reply:
BTW, I also wonder whether we really want that `--sort=no` ? It's there because `dev/ctags.readme.txt` suggests it, but I don't know why.

Reply:
The idea of the -F was to get both tag file formats in different files, but you're obviously right about names. I'd be happy with two targets, or with a different name for the emacs tag file (I can configure emacs to load it easily enough).

Reply:
You wrote `F` not `-F`, which confused me (and actually, the option is `-f`, while `-F` is something else, and ignored in `etags` mode, which confused me further :-).

I'd suggest we merge this PR as-is, and somebody else can add an `etags` target, or a second line to the `tags` target which deals with this -- I don't feel comfortable adding this as I wouldn't know how to test it.

Finally, we actually now have subdirs in `src` and `lib`, and may have more in the future; plus there is also `hpcgap/lib` (I think it's safe to ignore `hpcgap/src`, which only contains generated files). So I propose we drop the `--sort=no` and use `--recurse`, so: `ctags --recurse lib hpcgap/lib src`